cocos/ui/scroll-view.ts

```diff
@@ -880,11 +900,16 @@ export class ScrollView extends ViewGroup {
      * @zh
      * 获取当前视图内容的坐标点。
      *
-     * @returns - 当前视图内容的坐标点.
+     * @returns - current content position.
+     * @deprecated Since 3.0.1, getContentPosition is deprecated.
```

Comment:
Never deprecate any API in minor version.
Also what's the substitute of getContentPosition?

Reply:
I rebased the pr to develop branch, which should be released in v3.1